openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1108,7 +1109,11 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
 
 ## Batch Credential Response {#batch-credential-response}
 
-A successful Batch Credential Response MUST contain all the requested Credentials. The Batch Credential Response MUST be sent as a JSON object using the `application/json` media type.
+A successful Batch Credential Response MUST contain all the requested Credentials.
+
+If the Client requested an encrypted response, the Batch Credential Response MUST be sent as a JWT using the parameters from the `credential_response_encryption` object and using the `application/jwt` media type. If encryption was requested in the Batch Credential Request and the Batch Credential Response is not encrypted, the Client SHOULD reject the Credential Response.
```

Comment:
why is it `the Client SHOULD reject the Credential Response.` and not a MUST?

Reply:
I tried to keep it consistent with the Credential Response which currently says the same